data.sql

```diff
@@ -18,4 +18,44 @@ VALUES ('Charmander','2020/02/08',0,'false',-11,'Animal'),
        ('Angemon', '2005/06/12', 1, 'true', -45, 'Animal'),
        ('Boarmon', '2005/06/07', 7, 'true', 20.4, 'Animal'),
        ('Blossom', '1998/10/13', 3, 'true', 17, 'Animal'),
-       ('Ditto', '2022/05/14', 4, 'true', 22, 'Animal');
\ No newline at end of file
+       ('Ditto', '2022/05/14', 4, 'true', 22, 'Animal');
+
+/*  INSERT DATA INTO owners  */
+
+/*  Insert the following data into the owners table   */
+
+vet_clinic=# INSERT INTO owners (full_name,age)
+vet_clinic-# VALUES ('Sam Smith',19),
```

Comment:
Great job @Zven94 🤝

- [ ] Kindly ensure you have added the correct age value for Sam Smith. From the project requirements, it is supposed to be `34` instead of `19`